docs/changelog.rst

```diff
@@ -113,6 +116,9 @@ v0.20.0 (UNRELEASED)
 - Switch the ``list`` command over to using
   :meth:`mopidy.core.LibraryController.get_distinct`. (Fixes: :issue:`913`)
 
+- Add support for ``toggleoutput`` command. ``mixrampdb`` and ``mixrampdelay``
+  have also been added but have not been implemented yet.
```

Comment:
"The ``mixrampdb`` and ``mixrampdelay`` commands are now supported but throw a ``NotImplemented`` exception"?